src/ctapipe/image/muon/ring_fitter.py

```diff
@@ -32,22 +51,48 @@ class MuonRingFitter(Component):
 
     fit_method = traits.CaselessStrEnum(
         list(FIT_METHOD_BY_NAME.keys()),
-        default_value=list(FIT_METHOD_BY_NAME.keys())[0],
+        default_value="kundu_chaudhuri_taubin",
     ).tag(config=True)
 
-    def __call__(self, fov_lon, fov_lat, img, mask):
-        """allows any fit to be called in form of
-        MuonRingFitter(fit_method = "name of the fit")
+    def __call__(self, geom, image, clean_mask):
         """
+        Allows any fit to be called in form of
```

Comment:
This is more a docstring for the class itself, not for the `__call__` method.

I'd write something like:

```
Perform a circle fit to ``image`` with the chosen ``fit_method``.
```